ta4j-core/src/test/java/org/ta4j/core/indicators/CachedIndicatorTest.java

```diff
@@ -163,4 +163,26 @@ public void recursiveCachedIndicatorOnMovingTimeSeriesShouldNotCauseStackOverflo
             fail(t.getMessage());
         }
     }
+
+    @Test
+    public void leaveLastBarUncached() {
+        TimeSeries timeSeries = new MockTimeSeries(numFunction);
+        ClosePriceIndicator closePrice = new ClosePriceIndicator(timeSeries);
+        assertEquals(5000, closePrice.getValue(timeSeries.getEndIndex()).doubleValue(),0.001);
```

Comment:
Please use` TestUtils.assertNumEquals(expected, actual)` to allow comparison of current `Num` implementations and to avoid double transformation